hoodie-client/src/main/java/com/uber/hoodie/config/HoodieWriteConfig.java

```diff
@@ -58,6 +58,8 @@
   private static final String DEFAULT_ASSUME_DATE_PARTITIONING = "false";
   private static final String HOODIE_WRITE_STATUS_CLASS_PROP = "hoodie.writestatus.class";
   private static final String DEFAULT_HOODIE_WRITE_STATUS_CLASS = WriteStatus.class.getName();
+  private static final String INPUT_RECORD_STORAGE_LEVEL = "hoodie.input.record.storage.level";
+  private static final String DEFAULT_INPUT_RECORD_STORAGE_LEVEL = "MEMORY_AND_DISK_SER";
```

Comment:
This must be a BloomIndex level config, since other indexes won't respect it anyway. lets move to HoodieIndexConfig

lets rename to `hoodie.bloom.index.input.storage.level` and relevant methods as well

Reply:
Good point, will update.